npm/generate_docs.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// @ts-check
+
+import { existsSync, mkdirSync, writeFileSync } from "node:fs";
+import { dirname, join } from "node:path";
+import { fileURLToPath } from "node:url";
+
+import { generate_docs } from "./lib/node/qsc_wasm.cjs";
+
+const scriptDirPath = dirname(fileURLToPath(import.meta.url));
+const docsDirPath = join(scriptDirPath, "docs");
+
+if (!existsSync(docsDirPath)) {
+  mkdirSync(docsDirPath);
+}
+
+// 'filename' will be of the format 'namespace/api.md' (except for 'toc.yaml')
+// 'contents' will contain the full markdown expected
+
+/** @type {Array<{filename: string; contents: string}>} */
+const docs = generate_docs();
+
+if (!docs || !docs.length) {
+  throw new Error("No docs generated");
+}
+
+var today = new Date();
+var dd = String(today.getDate()).padStart(2, "0");
+var mm = String(today.getMonth() + 1).padStart(2, "0"); //January is 0!
+var yyyy = today.getFullYear();
+var today_str = mm + "/" + dd + "/" + yyyy + "12:00:00 AM";
```

Comment:
So there is no separator between yyyy and "12:00:00"? Does this look like "03/01/202412:00:00 AM" then? Doesn't seem right.

Reply:
Yep you are right, there should be a space. Fixed.